lib/smart_answer_flows/pay-leave-for-parents.rb

```diff
@@ -737,8 +733,8 @@ def define
                 end
               end
             end
-          elsif calculator.employment_status_of_mother == 'worker'
-            if calculator.employment_status_of_partner == 'employee'
+          elsif calculator.employment_status_of_mother == "worker"
+            if calculator.employment_status_of_partner == "employee"
               if calculator.partner_continuity?
                 if calculator.earnings_employment(calculator.partner_earned_at_least_390, calculator.partner_worked_at_least_26_weeks)
                   outcome :outcome_mat_pay_pat_leave_pat_shared_leave_mat_shared_pay
```

Comment:
Oh my god - this file is incredible.

What a block: https://github.com/alphagov/smart-answers/blob/2132cba9d19511fc579f7176d419318913678b51/lib/smart_answer_flows/pay-leave-for-parents.rb#L241-L642

It's a shame we don't have the linter on for cyclomatic complexity.